api/graphql/makeModels.js

```diff
@@ -202,6 +207,23 @@ export default function makeModels (userId, isAdmin) {
       model: Comment,
       attributes: ['id', 'created_at'],
       relations: [{user: {alias: 'creator'}}]
+    },
+
+    Vote: {
+      model: Vote,
+      attributes: [
+        'id',
+        'date_voted'
+      ],
+      relations: [
+        'post',
+        { user: { alias: 'voter' } }
+      ],
+      filter: nonAdminFilter(q => {
+        q.where('votes.post_id', 'in', PostMembership.query().select('post_id')
+          .where('community_id', 'in', myCommunityIds()))
+      }),
+      isDefaultTypeForTable: true
```

Comment:
you should be able to omit this because there isn't a different GraphQL type that also uses the Vote model